amlip_py/amlip_py/node/ModelManagerReceiverNode.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""AML-IP Model Receiver Receiver Node API specification."""
+
+
+from amlip_py.types.AmlipIdDataType import AmlipIdDataType
+from amlip_py.types.ModelDataType import ModelDataType
+from amlip_py.types.ModelSolutionDataType import ModelSolutionDataType
+from amlip_py.types.ModelStatisticsDataType import ModelStatisticsDataType
+
+from amlip_swig import ModelManagerReceiverNode as cpp_ModelManagerReceiverNode
+from amlip_swig import ModelListener as cpp_ModelListener
+
+
+class ModelListener(cpp_ModelListener):
+    """
+    Model Listener class.
+
+    This object must execute statistics_received method with each ModelStatisticsDataType that
+    is received from node and must return true or false. This object must also execute
+    model_received method on each ModelSolutionDataType that is received from node.
```

Comment:
This sounds like the listener object must perform the execution action. It is the manager node that calls the method.

Reply:
Same for the sender node listener.